From 3cb24463cad7aecd0dac2c0d283efeead1024050 Mon Sep 17 00:00:00 2001
From: rramakax <rohitx.kant@intel.com>
Date: Fri, 21 Aug 2020 13:38:40 +0530
Subject: [PATCH] CtsSensorTestCases are failing while running

CTS on CML device.

Sensors are enable from mixins spec but
CML wont have sensors.
Disabling the sensors to skip/pass
sensors related test cases on CML device.

Change-Id: If5285773511c4102a896a8aea747305b1a4014c0
Tracked-On: OAM-92321
Signed-off-by: rramakax <rohitx.kant@intel.com>

diff --git a/caas/mixins.spec b/caas/mixins.spec
index 2f4ac94..e914f21 100755
--- a/caas/mixins.spec
+++ b/caas/mixins.spec
@@ -83,4 +83,4 @@ atrace: true
 firmware: true(all_firmwares=true)
 aaf: true
 suspend: auto
-sensors: mediation
+sensors: false
-- 
1.9.1

